flink-connector-mysql-cdc/src/main/java/com/ververica/cdc/connectors/mysql/source/reader/MySqlSplitReader.java

```diff
@@ -121,6 +121,7 @@ private void checkSplitOrStartNext() throws IOException {
 
             if (nextSplit.isSnapshotSplit()) {
                 if (currentReader == null) {
+                    LOG.info("Create snapshot reader");
```

Comment:
This log is too simple.

We should know why we need this log and what information should be logged. I think Debezium does a good job for adding logs. You can take Debezium as an exmaple when not sure when/how/what to log.

Reply:
The log is to tell it's starting of snapshot phase and is ready to read snapshot splits.  Also it's  to be consistent with the log "Create binlog reader" in line 140.
Maybe it's no need to add this log.